services/movie.py

```diff
@@ -0,0 +1,40 @@
+from django.db.models import QuerySet
+
+import init_django_orm  # noqa: F401
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids
+        ).filter(actors__id__in=actors_ids)
```

Comment:
When filtering by both `genres_ids` and `actors_ids`, using consecutive `filter` calls might lead to unexpected results because it applies an AND condition across the filters. Consider using `Q` objects to combine these conditions if you intend to apply an OR condition or ensure the logic matches your requirements.

Reply:
Agree

Reply:
using Q causes one of the tests to fail